### PR TITLE
IntelliSense/Resource fixes

### PR DESCRIPTION
**Pull request by rowanmiller**
Fixing some incorrect api docs and string resources

### CODE REVIEW

File: src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx [hunk 1]
```diff
@@ -226,7 +226,7 @@
     <value>Root namespace of the project is required to generate code.</value>
   </data>
   <data name="MigrationsAssemblyMismatch" xml:space="preserve">
-    <value>Your target project '{assembly}' doesn't match your migrations assembly '{migrationsAssembly}'. Change your target project to the migrations project by using the Package Manager Console's Default project drop-down list or by using the '--targetProject' option for DNX commands. Change your migrations assembly by using DbContextOptionsBuilder. E.g. options.UseSqlServer(connection, b =&gt; b.MigrationsAssembly("{assembly}"))</value>
+    <value>Your target project '{assembly}' doesn't match your migrations assembly '{migrationsAssembly}'. Change your target project to the migrations project by using the Package Manager Console's Default project drop-down list or by using the '--targetProject' option for dotnet commands. Change your migrations assembly by using DbContextOptionsBuilder. E.g. options.UseSqlServer(connection, b =&gt; b.MigrationsAssembly("{assembly}"))</value>
```
Comment (rowanmiller) [inline src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx]: @natemcmaster what should this say for `dotnet ef`... how do I tell it what project to put the migrations into?

**Comment by ajcvickers** [comment]
:+1:

**Comment by ajcvickers** [comment]
That thumb is what CodeFlow posted when I click "Approve", So, :shipit:

File: src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx (hunk 1 above)
Comment (natemcmaster) [inline src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx]: `or by using the '--targetProject' option for dotnet commands` unfortunately this doesn't exist in CLI commands.

Comment (natemcmaster) [inline src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx]: > what should this say for dotnet ef... how do I tell it what project to put the migrations into?

@rowanmiller unfortunately this isn't really an option on CLI commands. The user needs to execute dotnet-ef on the project where they want migrations store. If this project is a library, they have to manually figure out how to move migrations between folders, setup different TFMs, etc....this is messy, and is a result of the limitations in CLI https://github.com/dotnet/cli/issues/2645.

Comment (rowanmiller) [inline src/Microsoft.EntityFrameworkCore.Tools.Core/Properties/ToolsCoreStrings.resx]: So how do people tell the command to put the migrations in another project?

**Pull request merged**